lib/api/middleware/set-linz-model.js

```diff
@@ -32,4 +60,7 @@ const setLinzModel = () => (req, res, next) => {
 
 };
 
-module.exports = setLinzModel;
+module.exports = {
+    getModelFromRequest,
```

Comment:
@allanchau, we shouldn't expose this if it's not intended for public use. Seeing as though `setLinzModel` is public, and it uses `getModelFromRequest`, can you just test via that?